lib/Stripe.php

```diff
@@ -199,12 +199,13 @@ public static function getAppInfo()
      * @param string $appVersion The application's version
      * @param string $appUrl The application's URL
      */
-    public static function setAppInfo($appName, $appVersion = null, $appUrl = null)
+    public static function setAppInfo($appName, $appPartnerId = null, $appUrl = null, $appVersion = null)
```

Comment:
Unfortunately PHP doesn't support keyword arguments, so this is a breaking change. Mind setting `$appPartnerId` as the last argument?